torchvision/datasets/mnist.py

```diff
@@ -249,13 +249,13 @@ class EMNIST(MNIST):
     md5 = "58c8d27c78d21e728a6bc7b3cc06412e"
     splits = ('byclass', 'bymerge', 'balanced', 'letters', 'digits', 'mnist')
     # Merged Classes assumes Same structure for both uppercase and lowercase version
-    _merged_classes = set(['C', 'I', 'J', 'K', 'L', 'M', 'O', 'P', 'S', 'U', 'V', 'W', 'X', 'Y', 'Z'])
-    _all_classes = set(list(string.digits + string.ascii_letters))
+    _merged_classes = {'c', 'i', 'j', 'k', 'l', 'm', 'o', 'p', 's', 'u', 'v', 'w', 'x', 'y', 'z'}
```

Comment:
One remark concerning cosmetics, not actual functionality. The [EMNIST paper](https://arxiv.org/pdf/1702.05373v1.pdf) describes removing the capital letters. We can actually write the classes in a way that achieves this but then the labels will look weird; something like: `A, B, c, D, ...`. So I think @vballoli's solution is fine.

Reply:
Yeah, I think this is fine as is, but thanks for pointing it out